netbench/run-netbench-request-response.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# build the scenario generation tool
+cargo build --manifest-path netbench-scenarios/Cargo.toml
+
+# generate the scenario files. This will generate .json files that can be found
+# in the netbench/target/netbench directory. Specify that there should be 1000
+# request-response occurrences
+cargo run --manifest-path netbench-scenarios/Cargo.toml -- --request_response.count 1000
+
+# build the drivers
+cargo build --manifest-path netbench-driver/Cargo.toml --release
+
+# build the statistics collectors
+cargo build --manifest-path netbench-collector/Cargo.toml --release
+
+# build the netbench cli
+cargo build --manifest-path netbench-cli/Cargo.toml --release
+
+# make a directory to hold the collected statistics
+mkdir -p results/request_response/s2n-quic/
+
+# run the server while collecting metrics. Generic Segmenetation Offload (GSO)
+# is disabled, because it results in production-disssimilar behaviors when
+# running over the loopback interface. Specifically, packets larger than the
+# max supported MTU can be observed.
```

Comment:
Spoke offline: this is to be expected. We hand the OS massive payload buffers and it defers splitting it up into individual packets for as long as possible. I think it's best to test with it enabled.